book/website/communication/open/curriculum-development.md

```diff
@@ -0,0 +1,233 @@
+*Figure 2: Bloom’s Taxonomy List of Verbs for creating Learning outcomes (Curriculum/Lesson)*
+
+## 2. Types of Sessions
+
+
+> Any learning, especially Science learning, can be divided into two parts: Theoretical and Practical. It is essential to make both types of sessions engaging for the students to enhance learning, it can be done by incorporating a variety of interactive and dynamic teaching methods, such as multimedia integration, interactive techniques, coding demonstration, personalisation, group work and signposting the lecture contents.
+### 
+### Theoretical Sessions
+Creating a good theoretical knowledge base for any topic is essential for effective learning and their implementation. This process has the following key components:
+
+1. **Developing the theory**: Establishing a theoretical foundation is important for Data Scientists to develop their skills to solve real world problems. Introduce fundamental concepts and theories related to the topic, ensuring that the learners grasp the essential principles before moving to more complex concepts. The explanation should be clear and concise, supported by relevant examples preferably real world to which the students can relate to. These concepts can be demonstrated through a dataset that is provided at the start of the course and examined when each component is being studied. These datasets can be opensource data or mock data that is generated specifically for this purpose.
+2. **Signposting Advanced Resources**: Students can have different levels of learning and interests on a topic. Signposting them towards relevant resources e.g. academic papers, books, online resources or expert talks is useful. It is essential to point the students towards reputable resources or provide a curated list for the whole course while dividing it into themes/topics that align with the course contents.
+
+### Practical Sessions
+To make learning more effective and interactive demonstrating the application of the theoretical concepts is useful, designing the practical sessions according to the demographics, capabilities and background of the students is essential. Practical sessions should be integrated into the curriculum and lesson design. 
+
+There are different teaching methdos that can be used to deliver practical sessions (Table 1). For example, for shorter sessions (1-hour) you can use **[flipped learning](https://www.advance-he.ac.uk/knowledge-hub/flipped-learning-0)** to maximise the efficiency of the learning process. Choosing flipped learning would depend on the demographics and preference of a cohort. Depending on the feasibility and preferences various forms of study material can be provided e.g. pre-recorded short lecture, reading material, provision of skeleton code that will be discussed or modified during the session etc. The in-class time should be dedicated to discussion, problem-solving, Q&A and hands-on activities to make it more engaging.
+<div class="alert alert-block alert-success">
```

Comment:
So excited to see this pull request, Denise! Congrats! 

I'll do a proper review in the coming days, but just wanted to flag first that we don't use the html way of formatting on our pages, and JupyterBook renders alerts and highlighted information using particular formats. You can see more in our styleguide: https://book.the-turing-way.org/community-handbook/style.html